lib/question.js

```diff
@@ -1888,29 +1888,35 @@ module.exports = {
                 submission.formatted_date = formatted_date;
                 submission.grading_job_stats = module.exports._buildGradingJobStats(grading_job);
 
-                const renderParams = {
-                  course,
-                  course_instance,
-                  question,
-                  submission,
-                  submissionHtml: htmls.submissionHtmls[0],
-                  submissionCount: submission_count,
-                  expanded: true,
-                  urlPrefix,
-                  plainUrlPrefix: config.urlPrefix,
-                };
-                const templatePath = path.join(
-                  __dirname,
-                  '..',
-                  'pages',
-                  'partials',
-                  'submission.ejs'
-                );
-                ejs.renderFile(templatePath, renderParams, (err, html) => {
-                  if (ERR(err, callback)) return;
-                  panels.submissionPanel = html;
-                  callback(null);
-                });
+                markdown
+                  .processContent(submission.feedback?.manual?.toString() || '')
+                  .then((html) => {
+                    submission.feedback_manual_html = html;
+
+                    const renderParams = {
+                      course,
+                      course_instance,
+                      question,
+                      submission,
+                      submissionHtml: htmls.submissionHtmls[0],
+                      submissionCount: submission_count,
+                      expanded: true,
+                      urlPrefix,
+                      plainUrlPrefix: config.urlPrefix,
+                    };
+                    const templatePath = path.join(
+                      __dirname,
+                      '..',
+                      'pages',
+                      'partials',
+                      'submission.ejs'
+                    );
+                    ejs.renderFile(templatePath, renderParams, (err, html) => {
+                      if (ERR(err, callback)) return;
+                      panels.submissionPanel = html;
+                      callback(null);
+                    });
+                  });
```

Comment:
We should handle any errors encountered while rendering Markdown:

```suggestion
                  }, (err) => ERR(err, callback));
```